sqlbrite/src/androidTest/java/com/squareup/sqlbrite/BriteDatabaseTest.java

```diff
@@ -546,6 +546,16 @@
     o.assertErrorContains("Cannot subscribe to observable query in a transaction.");
   }
 
+  @Test public void querySubscribedToDuringTransactionThrowsWithBackpressure() {
+    o.doRequest(0);
```

Comment:
This test won't actually fail without the change until `just` supports backpressure (which is how `startWith` is implemented)